cg/meta/invoice.py

```diff
@@ -8,59 +8,80 @@
 
 class InvoiceAPI():
 
-    def __init__(self, db: Store, lims_api: lims.LimsAPI):
+    def __init__(self, db: Store, lims_api: lims.LimsAPI, invoice_obj: models.Invoice):
```

Comment:
I see your point. Didnt think of that the class was called API. I thought of it more as an Invoice class. With functions to generate whats needed to build one invoice. Then it made more sense to define the invoice model instance in the init.  I was actually thinking of putting even mote things into the init, instead of retuning bunches of stuff from all the functions. But then I thought I shouldnt spend to much time on code finish. 

So after this explanation, you want me to take away the definition of the invoice model instance from the init? Or you want me to rename the class from InvoiceAPI to Invoice? Or i should just leave it as it is?

Im open for anything :)